### ISSUE TITLE
A11y View should match the displayed number of decimals

### ISSUE THREAD

**Issue by vperezcadavid99** [phetsims/alpha-decay#46]
**Test device**
Chromebook
**Operating System**
ChromeOs
**Browser**
Chrome
**Problem description**
From https://github.com/phetsims/qa/issues/1414

During the Multiple Atoms screen, the Half-life value is displayed with two decimal places. It would be better if the A11y View matched the same number of decimals.

<img width="2652" height="508" alt="Image" src="https://github.com/user-attachments/assets/0170a33d-4325-4156-b8f6-aa134737fae7" />

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Alpha Decay‬
URL: https://phet-dev.colorado.edu/html/alpha-decay/1.0.0-dev.4/phet/alpha-decay_all_phet.html
Version: 1.0.0-dev.4 2026-07-09 00:01:34 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/150.0.0.0 Safari/537.36
Language: en-US
Window: 1600x812
Pixel Ratio: 1.7999999523162842/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}
<details/>

**Comment by AgustinVallejo** [comment]
I believe this was fixed on another commit. Assigning back in case you saw another case, and if good, please close. Gracias!

**Comment by vperezcadavid99** [comment]
Thanks! It looks good for Polonium-211. 
I noticed that in Custom mode, the values don't use the same number of decimal places.

<img width="1576" height="208" alt="Image" src="https://github.com/user-attachments/assets/8febb6d6-aa02-4eaa-9163-e47ece5db03b" />

**Comment by AgustinVallejo** [comment]
Good catch, I've improved that consistency

**Comment by vperezcadavid99** [comment]
Gracias @AgustinVallejo !
Looks much better. Closing!
